ml-agents/mlagents/trainers/buffer.py

```diff
@@ -289,7 +269,12 @@ def reset_local_buffers(self):
             self[k].reset_agent()
 
     def append_update_buffer(
-        self, agent_id, key_list=None, batch_size=None, training_length=None
+        self,
```

Comment:
This name feels more misleading now - it sounds like you're taking the `update_buffer` and appending it to `self`, but it's actually the opposite.

How about either renaming to `append_to_update_buffer`, or make `append_processing_buffer` a method of `AgentBuffer`? The latter is a bigger change but I think it's a bit cleaner.

Reply:
(if nothing else, `append_to_update_buffer` is kinda close to `append_all_agent_batch_to_update_buffer`)

Reply:
`append_to_update_buffer` is pretty close. Eventually this method will be replaced by `add_trajectory_to_queue` so it probably shouldn't live in the low-level `AgentBuffer`.

Reply:
`Done()`

Reply:
SG. 

When I meant "append_to_update_buffer" is close to "append_all_agent_batch_to_update_buffer", I meant it in a good way (more consistent), not that they'd be mistaken for each other.

Reply:
Oh yeah, that makes sense 😄 Suppose it's technically `append_agent_trajectory_to_update_buffer`